product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/01_dashboards/17_str_replication_dashboard.mdx

```diff
@@ -42,9 +42,10 @@ When accessing the `Streaming Replication Analysis` dashboard for the replica no
 
 The `WAL Archive Status` graph displays WAL activity; the vertical key on the left side of the graph indicates the archive count; times are displayed across the bottom of the graph.
 
-The `WAL Segment Lag` graph displays the segment lag for the replica nodes that are associated with the selected server. The vertical key on the left side of the graph indicates the archive count. Each replica node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node.
+The `WAL Segment Lag` graph displays the segment lag for the replica nodes that are associated with the selected server. The vertical key on the left side of the graph indicates the archive count. Each replica node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node. Please note this data will only be displayed against the replica if it is replicating to another replica through cascading replication.
+
+The `WAL Page Lag` graph displays the page lag activity for each replica node associated with the selected server. The vertical key on the left side of the graph indicates the page count. Each node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node. Please note this data will only be displayed against the replica if it is replicating to another replica through cascading replication.
```

Comment:
```suggestion
The `WAL Page Lag` graph displays the page lag activity for each replica node associated with the selected server. The vertical key on the left side of the graph indicates the page count. Each node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node. This graph is only displayed on servers which are being replicated, and not on the replica nodes themselves unless these too are being replicated (i.e. cascading replication).
```

Reply:
Alternative form of words suggested. Whatever we choose we should apply to both places. I am trying to avoid too much of a tongue twister around replicating replicas to replicas. I am also trying to avoid placing undue focus on cascading replication. A terser, and still totally accurate, form would be "This graph is only displayed on servers which are being replicated", the rest is really just to help readers understand what that means.

Reply:
@drothery-edb what do you think?

Reply:
@sonotley mayber change slave to standby. because we are getting away from that language